sdcm/utils/version_utils.py

```diff
@@ -414,6 +414,18 @@ def resolve_latest_repo_symlink(url: str) -> str:
     return resolved_url
 
 
+def transform_non_semver_scylla_version_to_semver(scylla_version: str):
+    # NOTE: as of May 2022 all the non-GA versions of Scylla are not semver-like and it is problem
```

Comment:
non-GA, we mostly call them release candidate

also I think the date here is a bit missleading, since this change in scylla versioning is long ago

Reply:
"as of May 2022" sounds like it started in this date :)